evm_trace/base.py

```diff
@@ -19,6 +19,10 @@ class TraceFrame(BaseModel):
     memory: List[HexBytes] = []
     storage: Dict[HexBytes, HexBytes] = {}
 
+    @validator("pc", "gas", "gas_cost", "depth", pre=True)
+    def validate_ints(cls, value):
```

Comment:
Does Pydantic not have an issue if someone creates a `TraceFrame` with a str for fields like `pc` or `gas`?

Reply:
It does! 
Unless you have it get handled in validator, like it is here. 

This happens on Nethermind clients, according to some comment in Bronwnie, so it doesn't hurt to have this validator